launchable/commands/record/session.py

```diff
@@ -33,7 +45,11 @@ def session(build_name):
 
         # what we print here gets captured and passed to `--session` in later commands
         click.echo("{}/{}".format(session_path, session_id))
-        return session_id
+
+        if save_session_file:
+            write_session(build_name, "{}/{}".format(session_path, session_id))
+        else:
+            return session_id
```

Comment:
This caught my eyes. Why does the function stop returning a value when `--session-file` option is specified? What does the exit value from a `click.option` option do wrt Click? Nothing?

Reply:
It was my previous mistake. It looks like nothing happens. I replace it with `click.echo`.